bluetooth/src/jsMain/kotlin/UUID.kt

```diff
@@ -17,10 +17,7 @@
 
 package com.splendo.kaluga.bluetooth
 
-actual data class UUID (val uuidString: String) {
-    val isValid: Boolean
-        get() = uuidString.isValidUUIDString()
-}
+actual data class UUID (val uuidString: String)
```

Comment:
Why it has to be expect/actual?

Reply:
On iOS it is `CBUUID` and on android `java.util.UUID`

Reply:
It's js, right!

Reply:
Right. We need to add actual JS implementation